src/core/api/public_api.ts

```diff
@@ -171,6 +167,13 @@ interface IBitrateEstimate {
   bitrate : number | undefined;
 }
 
+export type IStreamEvent = { data: IStreamEventData;
+                             start: number;
+                             end: number;
+                             onExit?: () => void; } |
+                           { data: IStreamEventData;
+                             start: number; };
```

Comment:
I re-define it here to make sure this is a sanitized version for the API and also so that editors don't automatically export the type from the streamEventsEmitter